.husky/commit-msg

```diff
@@ -1,4 +1,2 @@
-#!/usr/bin/env sh
-. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
Why was it needed before and why is it not needed anymore? Doesn't husky rely on it?

The same comment applies to the `pre-commit` hook.

Reply:
It was removed automatically when upgrading husky. Apparently no longer needed.